HISTORY.md

```diff
@@ -1,3 +1,12 @@
+unreleased
+==================
+  * deps: handlebars@4.5.3
+    - Fix security issues 
```

Comment:
This change list seems incomplete; there are more changes than just this in the bump.